EmailValidator/Validation/EmailValidation.php

```diff
@@ -21,6 +21,8 @@ public function isValid($email, EmailLexer $emailLexer);
     /**
      * Returns the validation error.
      *
+     * Be careful about no-error does not necessarily mean email was valid.
```

Comment:
`NoRFCWarningsValidation` [can be](https://github.com/issei-m/EmailValidator/blob/68aae522c42d8253f415480e11ba8938e984b343/EmailValidator/Validation/NoRFCWarningsValidation.php#L11-L11).
This class will have no error if RFC check is success but with warnings.